subgraphs/uniswap-v3/protocols/uniswap-v3/config/deployments/uniswap-v3-ethereum/configurations.ts

```diff
@@ -81,42 +81,42 @@ export class UniswapV3MainnetConfigurations implements Configurations {
   }
   getUntrackedPairs(): Bytes[] {
     return stringToBytesList([
-      "0x8fe8d9bb8eeba3ed688069c3d6b556c9ca258248", // Mulan/USDT
-      "0xfd9715a9f03678975b2e224c938fae8d481b09b2", // Aurora/wETH
-      "0x9663f2ca0454accad3e094448ea6f77443880454", // LUSD/wETH
-      "0x785d7ff551c9b62562c6767551ffa3e3e562b6ed", // SiFum/USDT
+      // "0x8fe8d9bb8eeba3ed688069c3d6b556c9ca258248", // Mulan/USDT
```

Comment:
same deal 👀

Reply:
oops, is this the buffer?

Reply:
This was the blacklist. I went ahead and removed these tokens.